crates/ruff_python_ast/src/binding.rs

```diff
@@ -265,6 +265,26 @@ pub enum BindingKind<'a> {
     SubmoduleImportation(SubmoduleImportation<'a>),
 }
 
+impl<'a> BindingKind<'a> {
+    pub fn qualified_name(&self) -> Option<&'a str> {
+        match self {
+            BindingKind::Importation(importation) => Some(importation.full_name),
+            BindingKind::FromImportation(importation) => Some(&importation.full_name),
+            BindingKind::SubmoduleImportation(importation) => Some(importation.name),
```

Comment:
I also considered making this a trait and implementing it on these three `BindingKind` variants.